src/components/call_out/_mixins.scss

```diff
@@ -13,7 +13,7 @@
   @if (map-has-key($ouiCallOutTypes, $type)) {
     $color: map-get($ouiCallOutTypes, $type);
     $backgroundColor: tintOrShade($color, 90%, 70%);
-    $foregroundColor: shadeOrTint(makeHighContrastColor($color, $backgroundColor), 0, 20%);
+    $foregroundColor: shadeOrTint(makeHighContrastColor($color, $backgroundColor), 0%, 20%);
```

Comment:
This is potentially a change that will require changes in OSD; I know we use `makeHighContrastColor()` over there, too.

Reply:
This is only for sass compiler's use. OSD uses a different sass compiler so shouldn't complain.